### PR TITLE
Update stomatal conductance documentation

### PR DESCRIPTION
**Pull request by serbinsh**
Adding information related to the addition of Medlyn et al 2011 (USO) representation

Reflects changes made by @Qianyuxuan in FATES PR: https://github.com/NGEET/fates/pull/608

### CODE REVIEW

**Comment by serbinsh** [comment]
Medlyn info based on tech document provided by @Qianyuxuan 

[Note.for.Medlyn.stomatal.conductance.model.in.FATES-LQY-0515.pdf](https://github.com/NGEET/fates-docs/files/4663549/Note.for.Medlyn.stomatal.conductance.model.in.FATES-LQY-0515.pdf)

**Comment by serbinsh** [comment]
@Qianyuxuan @walkeranthonyp there is a statement in the original docs under stomatal conductance that I left in thees changes but I am not sure i follow and wonder if it was a typo?

```
 e_i is the saturation vapor pressure (Pa) inside the leaf at the vegetation temperature conductance (\mumol m^{-2} s^{-1}) when A = 0 
```

https://fates-docs.readthedocs.io/en/latest/fates_tech_note.html#water-stress-on-gas-exchange

Do you happen to know what that is supposed to mean, namely the vapor pressure in the leaf at the "vegetation temperature conductance"?  Strange wording.  Maybe an accidental cobble together of sentences?

**Comment by serbinsh** [comment]
@Qianyuxuan ready for review.  I tried to squash commits but failed and then had additional mistakes/typos.  I think I got it back in order

**Comment by Qianyuxuan** [comment]
> @Qianyuxuan @walkeranthonyp there is a statement in the original docs under stomatal conductance that I left in thees changes but I am not sure i follow and wonder if it was a typo?
> 
> ```
>  e_i is the saturation vapor pressure (Pa) inside the leaf at the vegetation temperature conductance (\mumol m^{-2} s^{-1}) when A = 0 
> ```
> 
> https://fates-docs.readthedocs.io/en/latest/fates_tech_note.html#water-stress-on-gas-exchange
> 
> Do you happen to know what that is supposed to mean, namely the vapor pressure in the leaf at the "vegetation temperature conductance"? Strange wording. Maybe an accidental cobble together of sentences?

Yeah, I think so.

**Comment by serbinsh** [comment]
> > @Qianyuxuan @walkeranthonyp there is a statement in the original docs under stomatal conductance that I left in thees changes but I am not sure i follow and wonder if it was a typo?
> > ```
> >  e_i is the saturation vapor pressure (Pa) inside the leaf at the vegetation temperature conductance (\mumol m^{-2} s^{-1}) when A = 0 
> > ```
> > 
> > 
> > https://fates-docs.readthedocs.io/en/latest/fates_tech_note.html#water-stress-on-gas-exchange
> > Do you happen to know what that is supposed to mean, namely the vapor pressure in the leaf at the "vegetation temperature conductance"? Strange wording. Maybe an accidental cobble together of sentences?
> 
> Yeah, I think so.

Sorry that was the wrong link, here is the sentence in the docs I am unsure about

A is leaf photosynthesis (\mumol CO_2 m^{-2} s^{-1}), c_s is the CO_2 partial pressure at the leaf surface (Pa), e_s is the vapor pressure at the leaf surface (Pa), e_i is the saturation vapor pressure (Pa) inside the leaf at the vegetation temperature conductance (\mumol m^{-2} s^{-1}) when A = 0 . 

Particularly "e_i is the saturation vapor pressure (Pa) **inside the leaf at the vegetation temperature conductance** (\mumol m^{-2} s^{-1}) when A = 0 . "

File: docs/source/fates_tech_note.rst
Comment (serbinsh) [inline docs/source/fates_tech_note.rst:2141]: Is it a bad idea to embed parameter defaults in the docs? Will require keeping in line with any changes in the params.  But helpful as it exposes these to the user. That said the text version of the param file is easy to view on Git.  So I am between two minds here

File: docs/source/fates_tech_note.rst
Comment (serbinsh) [inline docs/source/fates_tech_note.rst:2075]: @Qianyuxuan updated this portion based on our discussion and clarification of the previous confusing, erroneous text

Comment (Qianyuxuan) [reply docs/source/fates_tech_note.rst:2075]: @serbinsh Nice work! Looks great to me.

**Comment by serbinsh** [comment]
@rgknox @glemieux FYI - made some updates here and waiting for the final sign off from @Qianyuxuan.  But she is working on her manuscript and we would like to be able to cite these docs in her paper, including the contents of this PR.  As such, how should we think about merging doc changes?  For example, I believe her code changes are already in master now so we should probably also bring these changes in to make sure docs match code.  Is this something we can ask once she is sure its all good? Do you plan to wait for a few more PRs?

Thanks!

CC @alistairrogers

**Comment by glemieux** [comment]
@serbinsh thanks for the heads up.  I'll talk with @rgknox and @ckoven about pushing a new release to update the DOI after merging these updates.

**Comment by glemieux** [comment]
@Qianyuxuan do the changes that @serbinsh made look good to you?  If so, is it ok if I integrate this pull request now or are there more changes forthcoming I should wait for?  Thanks.

**Comment by Qianyuxuan** [comment]
Sorry I just noticed it. I have reviewed the changes before and they look nice. I think it’s ready for integration. Thanks!
Best,
Cherry

**Review by glemieux — APPROVED** [review]

**Comment by serbinsh** [comment]
Awesome, thank you @glemieux!

**Pull request merged**
